src/client/dfs/SConscript

```diff
@@ -18,18 +18,25 @@ def scons():
                     break
         if distro in ['centos', 'rhel', 'fedora', 'opensuse']:
             import rpm
-            from pkg_resources import parse_version
 
+            gotit = False
             ts = rpm.TransactionSet()
             headers = ts.dbMatch('name', 'lustre')
             if not headers:
                 headers = ts.dbMatch('name', 'lustre-client')
-            if headers and parse_version(headers['version']) >= parse_version('2.12.57'):
-                denv.AppendUnique(CCFLAGS=['-DLUSTRE_INCLUDE'])
-            else:
-                print("Found no package lustre nor lustre-client.")
+            if headers:
+                from pkg_resources import parse_version
+
+                for header in headers:
+                    if parse_version(header['version']) >= parse_version('2.12.57'):
+                        denv.AppendUnique(CCFLAGS=['-DLUSTRE_INCLUDE'])
+                        gotit = True
+            if gotit == False:
```

Comment:
(pylint-singleton-comparison) Comparison to False should be 'not expr' or 'expr is False'